frontend/src/static/js/v_summary.js

```diff
@@ -119,6 +124,9 @@ window.vSummary = {
       let totalCount = 0;
       this.repos.forEach((repo) => {
         repo.users.forEach((user) => {
+          if (user.checkedFileTypeContribution === undefined) {
```

Comment:
By issuing an extra check for the checkedFileTypeContribution property and defining it, the issue of missing contribution bars will be fixed. Even so, I'm not sure if its intentional for the checkedFileTypeContribution property to be undefined.

Reply:
You may try to found the root issue of why this property will be undefined here.

Reply:
`avgContributionSize` is computed based off `checkedFileTypeContribution` of users in this.repos ever since the change from commit 46c0d8e3d9957aa6aa440bff1440d7a1eee1e6a3. 

As far as I know, `avgContributionSize` only affects the UI of the contribution bar. What I'm unsure of is whether if it is intentional for `avgContributionSize` to vary according to the of the availability of the `checkedFileTypeContribution` property, and does the change from the commit assumes that all `checkedFileTypeContribution` are defined?

Reply:
@0blivious Can you help answer the question?

Reply:
Closer inspection seems to indicate that the computation of `avgContributionSize` helps to set the limit on the length of the contribution bar on initialization, but any future changes in checked file types does not lead to a recomputation of this limit.

Below is an example of what happens as this limit is not updated when file types are checked/unchecked
<img width="682" alt="image" src="https://user-images.githubusercontent.com/35420846/98821072-00979700-246a-11eb-870e-2b6ba47ac0a7.png">

In my opinion, we can stay with the current implementation as this property is not serving it's purpose yet. but we have to figure a new way on how we can recompute this limit whenever file types are checked/unchecked.